Source/JavaScriptCore/inspector/remote/cocoa/RemoteConnectionToTargetCocoa.mm

```diff
@@ -198,6 +198,8 @@ static void RemoteTargetHandleRunSourceWithInfo(void* info)
 void RemoteConnectionToTarget::close()
 {
     auto targetIdentifier = m_target ? m_target->targetIdentifier() : 0;
+    if (auto* automationTarget = dynamicDowncast<RemoteAutomationTarget>(m_target); m_connected)
```

Comment:
Weird syntax, but ok

Reply:
is checking for `m_connected` really even necessary in the first place?

Reply:
Good point Devin. Since all we are doing is setting a flag, we don't need to be concerned with the state like we do before calling disconnect. (This is what happens when I simultaneously manage to over-think and under-think 😆)